### ISSUE TITLE
It should be possible to save tracks to your library from more places

### ISSUE THREAD

**Issue by laurinneff** [xou816/spot#505]
It would be useful if it was possible to save tracks from playlists, and maybe we could also have a button in the now playing bar. An option in the 3 dot menu would also be nice, since it's annoying to have to switch to the multi-selection mode every time

**Comment by SarahRemus** [comment]
I'm going to start working on this issue.

**Comment by MFAshby** [comment]
> An option in the 3 dot menu would also be nice,

agreed, this is the first place i looked and it took me ages to find the actual option (having to multi-select first is very odd)
